webgoat-lessons/path-traversal/src/main/resources/lessonPlans/en/PathTraversal_upload_fix.adoc

```diff
@@ -0,0 +1,11 @@
+=== Path traversal while uploading files
+
+Again the same assignment but can you bypass the implemented fix?
```

Comment:
Repeat what the fix was on this page.

Reply:
And add the WebWolf pic to show you need an intercepting proxy

Reply:
Good idea, fixed. Not sure why the WebWolf pic is necessary? No functionality of WebWolf will be used I guess?

Reply:
it was mentioned in the notes on the first page, but indeed not necessary